src/components/ButtonPanel.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import Button from './Button';
+
+const ButtonPanel = () => (
+  <div>
+    <div className="row">
+      <Button buttonName="AC">AC</Button>
```

Comment:
Why are you repeating yourself here? :thinking:

Passing "AC" two times is redundant, pass it as props only, it will be rendered as such.

Reply:
I'll improve on this.